<?xml version="1.0"?><!DOCTYPE html PUBLIC "-//W3C//DTD XHTML+ARIA 1.0//EN" "http://www.w3.org/MarkUp/DTD/xhtml-aria-1.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en-GB">
<head>
		<title>Homepage of Petr Šulc</title>
		<meta name="description" content="" />
		<meta name="keywords" content="" />
		<meta http-equiv="content-language" content="en-GB" />
		<meta http-equiv="content-type" content="text/html; charset=utf-8" />
		<!--<link rel="shortcut icon" href="/favicon.ico" />-->

		<meta http-equiv="content-script-type" content="text/javascript" />
		<meta http-equiv="content-style-type" content="text/css" />
		<meta http-equiv="imagetoolbar" content="no" />
<meta http-equiv="cache-control" content="max-age=0" />
<meta http-equiv="cache-control" content="no-cache" />
<meta http-equiv="expires" content="0" />
<meta http-equiv="expires" content="Tue, 01 Jan 1980 1:00:00 GMT" />
<meta http-equiv="pragma" content="no-cache" />
		<meta name="robots" content="index,follow" />
		<meta http-equiv="Cache-Control" content="no-cache, no-store, must-revalidate">
		<meta http-equiv="Pragma" content="no-cache">
		<meta http-equiv="Expires" content="0">

        <meta http-equiv="content-script-type" content="text/javascript" />
        <meta http-equiv="content-style-type" content="text/css" />
        <meta http-equiv="imagetoolbar" content="no" />

        <meta name="copyright" content="&copy; 2010" />

		  <link rel="stylesheet" type="text/css" href="css/smoothness/jquery-ui-1.8.5.custom.css" />
        <link rel="stylesheet" type="text/css" href="css/main.css" />

        <script type="text/javascript" src="js/jquery-1.4.2.min.js"></script>
        <script type="text/javascript" src="js/jquery-ui-1.8.5.custom.min.js"></script>

        <script type="text/javascript">

        $(document).ready(function(){
           $('button.btp').button({
            icons: {primary: "ui-icon-arrowthickstop-1-n"}
            });

        });


        </script>
</head>

<body>

<p class="ui-helper-hidden-accessible">
<a href="#content-main" title="main content">main content</a>
</p>

<div id="wrapper-outer">

	<div id="header">

	<div id="tr-image">
	 <img src="images/logo.png" height="150" alt="Photograph"  width="120.5"  align=top  border="0" />
	
	</div>



	<h1>Petr Šulc's lab</h1>

	<div id="menu-top">
		<ul>
    <li class="generic"><a href="index.html">Home</a></li>
    <li class="generic"><a href="group.html">Members</a></li>
    <li class="generic"><a href="research.html">Research</a></li>
    <li class="generic"><a href="software.html">Software</a></li>
    <li class="generic inner current"><a href="publications.html">Publications</a></li>
    <li class="generic"><a href="bio.html">Petr Šulc</a></li>
    </ul>
	</div>

	</div>


	<div id="content-main">


<h2>Publications</h2>
Most recent list is available on a <a href="http://scholar.google.co.uk/citations?user=sUUmHhUAAAAJ&hl=en&oi=ao">Google Scholar profile</a>.
<br>
<ol>

<h3> 2025</h3>


	<li><a href="https://www.pnas.org/doi/10.1073/pnas.2416988122">Controlling DNA-RNA strand displacement kinetics with base distribution</a>  <br>
		PNAS (2025); bioRxiv  <a href="https://www.biorxiv.org/content/10.1101/2024.08.06.606789v1">preprint</a>  <br>
		Eryk Ratajczyk, Jonathan Bath, Petr Sulc, Jonathan Doye, Ard Louis, Andrew Turberfield
	

<li><a href="https://journals.aps.org/prl/abstract/10.1103/PhysRevLett.134.148201"> Falsifiability Test for Classical Nucleation Theory  </a>  <br>
	Phys. Rev. Lett. (2025);  	arxiv  <a href="https://arxiv.org/abs/2503.23124">preprint</a>  <br>
	Camilla Beneduce, Diogo EP Pinto, Lorenzo Rovigatti, Flavio Romano, Petr Šulc, Francesco Sciortino, John Russo


<li><a href="https://arxiv.org/abs/2505.10544"> High-endurance mechanical switching in a DNA origami snap-through mechanism
  </a>  <br>
	arxiv  <a href="https://arxiv.org/abs/2505.10544">preprint</a>  <br>
Florian Rothfischer, Lennart J. K. Weiß, Niccolò Tedeschi, Michael Matthies, Matthias Vogt, Christoph Karfusehr, Alexander Hebel, Petr Šulc, Tim Liedl, Enzo Kopperger, Friedrich C. Simmel

<li><a href="https://www.biorxiv.org/content/10.1101/2025.03.14.643268.abstract"> Mismatch-Assisted Toehold Exchange Cascades for Magnetic Nanoparticle-Based Nucleic Acid Diagnostics
</a>  <br>
	 	bioarxiv  <a href="https://www.biorxiv.org/content/10.1101/2025.03.14.643268.abstract">preprint</a>  <br>
		 Rebecca Sack, Joshua Evans, Florian Wolgast, Meinhard Schilling, Thilo Viereck, Petr Šulc, Aidin Lak



<li><a href="https://onlinelibrary.wiley.com/doi/10.1002/smtd.202401526">How we simulate DNA origami</a>  <br>
	Small Methods (2025);  	arxiv  <a href="https://arxiv.org/abs/2409.13206">preprint</a>  <br>
	Sarah Haggenmueller, Michael Matthies, Matthew Sample, Petr Šulc


<li><a href="https://pubs.acs.org/doi/10.1021/acsnano.4c10434"> Automating Blueprints for Colloidal Quasicrystal Assembly</a>  <br>
	ACS Nano (2025)
		 arxiv <a href="https://arxiv.org/abs/2407.19968">preprint</a>   <br>
	Diogo Pinto, Petr Šulc, Francesco Sciortino, John Russo




<h3> 2024</h3>

<li><a href="https://www.science.org/stoken/author-tokens/ST-1861/full"> Inverse design of a pyrochlore lattice of DNA origami through model-driven experiments
</a>  <br>
Science, 384, 6697, (741-742), (2024); arxiv <a href="https://arxiv.org/abs/2310.10995">preprint</a> <br>
covered by <a href="https://www.science.org/doi/10.1126/science.adp4370"> Perspective </a> in the same issue of Science<br>
Hao Liu, Michael Matthies, John Russo, Lorenzo Rovigatti, Raghu Pradeep Narayanan, Thong Diep, Daniel McKeen, Oleg Gang, Nicholas Stephanopoulos, Francesco Sciortino, Hao Yan, Flavio Romano, Petr Šulc


	

<li><a href="https://pubs.acs.org/doi/full/10.1021/acsnano.4c10796"> Hairygami: Analysis of DNA Nanostructures' Conformational Change Driven by Functionalizable Overhangs	</a>  <br>
	ACS Nano, 2024 <br>
	arxiv  <a href="https://arxiv.org/abs/2302.09109">preprint</a>  <br>
	Matthew Sample, Thong Diep, Hao Liu, Michael Matthies, Petr Šulc



<li><a href="https://pubs.acs.org/doi/10.1021/acscentsci.4c01428">Combination of Coevolutionary Information and Supervised Learning Enables Generation of Cyclic Peptide Inhibitors with Enhanced Potency from a Small Data Set	</a>  <br>
	ACS Central Science, 2024 <br>
	Ylenia Mazzocato, Nicola Frasson, Matthew Sample, Cristian Fregonese, Angela Pavan, Alberto Caregnato, Marta Simeoni, Alessandro Scarso, Laura Cendron, Petr Šulc, Alessandro Angelini



<li><a href="https://dl.acm.org/doi/10.1145/3703920"> Evolving to find optimizations humans miss: using evolutionary computation to improve GPU code for bioinformatics applications	</a>  <br>
	ACM Transactions on Evolutionary Learning (2024) <br>
	Jhe-Yu Liou, Muaaz Awan, Kirtus Leyba, Petr Šulc, Steven Hofmeyr, Carole-Jean Wu, Stephanie Forrest 


<li><a href="https://doi.org/10.1063/5.0202829">ANNaMo: Coarse-grained modeling for folding and assembly of RNA and DNA systems</a>  <br>
	J. Chem. Phys. 160, 205102 (2024); arxiv <a href="https://arxiv.org/abs/2311.03317">preprint</a>  <br>
	F. Tosti Guerra, E. Poppleton, P. Šulc, L. Rovigatti


<li><a href="https://www.nature.com/articles/s41565-023-01516-x"> A rhythmically pulsing leaf-spring nanoengine that drives a passive follower
</a>  <br>
 Nature Nanotechnology 19, 226-236 (2024) ; biorxiv <a href="https://www.biorxiv.org/content/10.1101/2021.12.22.473833v1"> preprint  </a><br>
M. Centola, E. Poppleton, M. Centola, J. Valero, S. Ray, R. Welty, Nils G. Walter, P. Šulc, M. Famulok





<li><a href="https://pubmed.ncbi.nlm.nih.gov/39104121/">Coarse-grained modeling of DNA-protein interactions helps elucidate DNA compaction
</a>  <br>
		New and Notable perspective for Biophysical journal  <br>
		Petr Sulc
	
	

<li><a href="https://pubs.aip.org/aip/jcp/article/160/11/115101/3275728/Coarse-grained-modeling-of-DNA-RNA-hybrids?searchresult=1"> Coarse-grained modelling of DNA-RNA hybrids</a>  <br>
	J. Chem. Phys. 160, 115101 (2024); article was featured in <a href="https://pubs.aip.org/aip/sci/article/2024/12/121101/3277576/Coarse-grained-model-simulates-DNA-RNA-hybrids-for">Scilight</a> ;  arxiv <a href="https://arxiv.org/abs/2311.07709">preprint</a>;  <br>
		Eryk J. Ratajczyk, Petr Šulc, Andrew J. Turberfield, Jonathan P.K. Doye, Ard A. Louis


<li><a href="https://journals.aps.org/prl/abstract/10.1103/PhysRevLett.132.118201"> Inverse design of self-folding 3D shells</a>  <br>
	Phys. Rev. Lett. 132, 118201  (2024) ; 	arxiv <a href="https://arxiv.org/abs/2310.03537">preprint</a>   <br>
	Diogo Pinto, Nuno Araújo, Petr Šulc, John Russo

<li><a href="https://journals.aps.org/prxlife/abstract/10.1103/PRXLife.2.013012"> RNA-induced allosteric coupling drives viral capsid assembly  </a>  <br>
	PRX Life (2024), article selected for spotlight in <a href="https://link.aps.org/doi/10.1103/Physics.17.s33">Physics Magazine</a>  <br>
	Sean Hamilton, Tushar Modi, Petr Šulc, Banu Ozkan


<li><a href="https://doi.org/10.1063/5.0191282">Designing 3D multicomponent self-assembling systems with signal-passing building blocks</a>  <br>
		J. Chem. Phys. 160, 084902 (2024): Emerging Investigators Special Collection; selected as "Editor's Pick" ; <a href="https://arxiv.org/abs/2312.13479">arxiv </a> preprint  <br>
		Joshua Evans, Petr Šulc


<li><a href="https://pubs.acs.org/doi/10.1021/acs.nanolett.3c04497"> DNA-Nanostructure-Guided Assembly of Proteins into Programmable Shapes</a>  <br>
	Nano Lett. (2024) <br>
	Qinyi Lu, Yang Xu, Erik Poppleton, Kun Zhou, Petr Sulc, Nicholas Stephanopoulos, Yonggang Ke


	

<li><a href="https://www.nature.com/articles/s41467-024-51813-9"> Single-Molecule Force Spectroscopy of Toehold-Mediated Strand Displacement
</a>  <br>
	Nature Communications 15 (2024); biorXiv <a href="https://www.biorxiv.org/content/10.1101/2024.01.16.575816v1.abstract">preprint</a> <br>
	Andreas Walbrun, Tianhe Wang, Michael Matthies, Petr Šulc, Friedrich Simmel, Matthias Rief


	
	
	
<h3> 2023</h3>

<li><a href="https://pubs.acs.org/doi/10.1021/jacs.3c07491"> CytoDirect: A Nucleic Acid Nanodevice for Specific and Efficient Delivery of Functional Payloads to the Cytoplasm
</a>  <br>
	JACS (2023) <br>
	Lu Yu, Yang Xu, Md Al-Amin, Shuoxing Jiang, Matthew Sample, Abhay Prasad, Nicholas Stephanopoulos, Petr Šulc, and Hao Yan

<li><a href="https://pubs.acs.org/doi/10.1021/acsnano.3c05569"> Engineering Azeotropy to Optimize the Self-Assembly of Colloidal Mixtures
</a>  <br>
	ACS Nano (2023); arxiv <a href="https://arxiv.org/abs/2208.09856"> preprint </a>  <br>
	Camilla Beneduce, Francesco Sciortino, Petr Šulc, John Russo


	

<li> <a href="https://www.biorxiv.org/content/10.1101/2023.09.18.558353v1.abstract">High-affinity binding to the SARS-CoV-2 spike trimer by a nanostructured, trivalent protein-DNA synthetic antibody
</a> <br>
biorxiv preprint (2023) <br>
Yang Xu, Rong Zheng, Abhay Prasad, Minghui Liu, Zijian Wan, Xiaoyan Zhou, Ryan M Porter, Matthew Sample, Erik Poppleton, Jonah Procyk, Hao Liu, Yize Li, Shaopeng Wang, Hao Yan, Petr Šulc, Nicholas Stephanopoulos



	<li> <a href="https://www.biorxiv.org/content/10.1101/2023.08.29.555281v1">High-speed 3D DNA-PAINT and unsupervised clustering for unlocking 3D DNA origami cryptography
	</a> <br>
	biorxiv preprint (2023) <br>
	G. Bimananda M. Wisna, Daria Sukhareva, Jonathan Zhao, Deeksha Satyabola, Michael Matthies, Subhajit Roy, Petr Šulc, Hao Yan, Rizal F. Hariadi
	
	
	

	<li> <a href="https://arxiv.org/abs/2308.01455">Coarse-grained simulations of DNA and RNA systems with oxDNA and oxRNA models: Introductory tutorial

	</a> <br>
	Proceedings of Winter Simulation Conference (2023) <br>
	 arxiv <a href="https://arxiv.org/abs/2308.01455">preprint</a> <br>
	 Michael Matthies, Matthew Sample, Petr Šulc

    <li><a href="https://pubs.acs.org/doi/abs/10.1021/jacs.3c03044"> DNA Origami Tessellations</a>  <br>
		J. Am. Chem. Soc., 145, 25 (2023)  <br>
		Yue Tang, Hao Liu, Qi Wang, Xiaodong Qi, Lu Yu, Petr Šulc, Fei Zhang, Hao Yan, Shuoxing Jiang
	


	<li><a href="https://pubs.acs.org/doi/abs/10.1021/acs.jcim.3c00358"> Atomistic Picture of Opening–Closing Dynamics of DNA Holliday Junction Obtained by Molecular Simulations	</a>  <br>
		Journal of Chemical Information and Modeling (2023)  <br>
		Zhengyue Zhang, Jiří Šponer, Giovanni Bussi, Vojtěch Mlýnský, Petr Šulc, Chad R Simmons, Nicholas Stephanopoulos, Miroslav Krepl
		


	<li><a href="https://doi.org/10.1073/pnas.2219458120"> Design strategies for the self-assembly of polyhedral shells	</a>  <br>
		Proceedings of the National Academy of Sciences 120 (16),  (2023)  <br>
		Diogo E. P. Pinto; Petr Šulc; Francesco Sciortino; John Russo


	
	<li><a href="https://pubs.acs.org/doi/10.1021/acsnano.2c09677"> Designing the self-assembly of arbitrary shapes using minimal complexity building blocks	</a>  <br> 
		ACS Nano (2023) (featured on the <a href="myimages/acs_cover.jpg">cover</a>) ;		arxiv <a href="https://arxiv.org/abs/2207.06954">preprint</a>   <br>
			Joakim Bohlin, Andrew Turberfield, Ard Louis, Petr Šulc
	

	<li><a href="https://doi.org/10.1063/5.0140847"> Two-step nucleation in a binary mixture of Patchy Particles	</a>  <br>
		J. Chem. Phys. (2023); selected as "Editor's pick"  <br>
		Camilla Beneduce, Diogo Pinto, Petr Šulc,  Francesco Sciortino, John Russo
	

	<li><a href="https://joss.theoj.org/papers/10.21105/joss.04693.pdf"> oxDNA: coarse-grained simulations of nucleic acids made simple	</a>  <br>
			Journal of Open Source Software (2023)  <br>
			Erik Poppleton, Michael Matthies, Debesh Mandal, Flavio Romano, Petr Šulc, Lorenzo Rovigatti
	
	
	
	<h3> 2022</h3>


	

	<li> <a href="https://www.sciencedirect.com/science/article/pii/S2001037022005037">Biomotors, viral assembly, and RNA nanobiotechnology: Current achievements and future directions</a> <br>
	   Computational and Structural Biotechnology Journal (2022) <br>
	   L. Rolband et al
	
	<li><a href="https://pubs.acs.org/doi/abs/10.1021/acsnano.2c04013"> Coarse-Grained Simulations for the Characterization and Optimization of Hybrid Protein–DNA Nanostructures	</a>  <br>
			ACS Nano (2022)  <br>
			Raghu Pradeep Narayanan, Jonah Procyk, Purbasha Nandi, Abhay Prasad, Yang Xu, Erik Poppleton, Dewight Williams, Fei Zhang, Hao Yan, Po-Lin Chiu, Nicholas Stephanopoulos, Petr Šulc
		
	
	<li><a href="https://www.nature.com/articles/s41596-022-00688-5"> Design and simulation of DNA, RNA and hybrid protein–nucleic acid nanostructures with oxView	</a>  <br>
		Nature Protocols (2022)  <br>
		Joakim Bohlin, Michael Matthies, Erik Poppleton, Jonah Procyk, Aatmik Mallya, Hao Yan, Petr Šulc
	
	<li><a href="https://www.nature.com/articles/s41467-022-30779-6"> The influence of Holliday junction sequence and dynamics on DNA crystal self-assembly	</a>  <br>
	Nature Communications 13 (2022)  <br>
	Chad R. Simmons, Tara MacCulloch, Miroslav Krepl, Michael Matthies, Alex Buchberger, Ilyssa Crawford, Jiří Šponer, Petr Šulc, Nicholas Stephanopoulos, Hao Yan


	<li><a href="https://doi.org/10.1039/D2NR03533B"> A simple solution to the problem of self-assembling cubic diamond crystals	</a>  <br>
		Nanoscale 14 (2022), arxiv <a href="https://arxiv.org/abs/2205.10680">preprint</a>  <br>
		Lorenzo Rovigatti, John Russo, Flavio Romano, Michael Matthies, Lukáš Kroc, Petr Šulc

	<li><a href="https://www.science.org/doi/full/10.1126/sciadv.abm9530"> A localized DNA finite-state machine with temporal resolution
	</a>  <br>
	Science Advances 8 (12) (2022)  <br>
	Lan Liu, Fan Hong, Hao Liu, Xu Zhou, Shuoxing Jiang, Petr Šulc, Jian-Hui Jiang, Hao Yan

	<li><a href="https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1010561"> Generative and interpretable machine learning for aptamer design and analysis of in vitro sequence selection
	</a>  <br>
	PLOS Comp. Biology (2022);  <br>
	Andrea Di Gioacchino,  Jonah Procyk,  Marco Molari,  John S. Schreck, Yu Zhou, Yan Liu,  Rémi Monasson,  Simona Cocco, Petr Šulc

	<li><a href="https://iopscience.iop.org/article/10.1088/1361-648X/ac5479/meta"> SAT-assembly: A new approach for designing self-assembling systems 	</a>  <br>
		Journal of Physics: Cond. Matter (2022); arxiv <a href="https://arxiv.org/abs/2111.04355">preprint</a> <br>
	 John Russo, Flavio Romano, Lukáš Kroc, Francesco Sciortino, Lorenzo Rovigatti, Petr Šulc	


	<li><a href="https://www.mdpi.com/1420-3049/27/1/63"> Unified Nanotechnology Format: One Way to Store Them All
	</a>  <br>
		Molecules, 27 (2022) <br>
		David Kuťák, Erik Poppleton, Haichao Miao, Petr Šulc, Ivan Barišić


	<li><a href="https://pubsonline.informs.org/doi/abs/10.1287/ijoc.2022.1188"> ExpertRNA: A new framework for RNA structure prediction
		</a>  <br>
		Informs Journal on Computing (2022);  bioRxiv <a href="https://www.biorxiv.org/content/10.1101/2021.01.18.427087v1.abstract">preprint</a>
		  <br>
		M. Liu, G. Pedrielli, E. Poppleton, P. Šulc, D. P. Bertsekas	
	
	<li><a href="https://doi.org/10.1093/nar/gkab1000"> Nanobase.org: a repository for DNA and RNA nanostructures </a>  <br>
		Nucleic Acids Research, (2022), article featured on the front <a href="coverfig.jpeg">cover </a> <br>
		Erik Poppleton, Aatmik Mallya, Swarup Dey, Joel Joseph, Petr Šulc

	<li><a href="https://www.sciencedirect.com/science/article/pii/S2589004222008719">Y-RNAs Lead an Endogenous Program of RIG-I Agonism Mobilized upon RNA Virus Infection and Targeted by HIV
		</a><br> iScience 2022 <br>
		Nicolas Vabret, Valérie Najburg, Alexander Solovyov, Ramya Gopal, Christopher McClain, Petr Šulc, Sreekumar Balan, Yannis Rahou, Guillaume Beauclair, Maxime Chazal, Hugo Varet, Rachel Legendre, Odile Sismeiro, Raul Y Sanchez David, Lise Chauveau, Nolwenn Jouvenet, Martin Markowitz, Sylvie van der Werf, Olivier Schwartz, Frédéric Tangy, Nina Bhardwaj, Benjamin D Greenbaum, Anastassia V Komarova
		</li>

	
	<h3> 2021</h3>

	<li><a href="https://www.biorxiv.org/content/10.1101/2021.11.04.467016v1"> Repeats Mimic Immunostimulatory Viral Features Across a Vast Evolutionary Landscape
		</a>  <br>
		bioRxiv preprint (2021)  <br>
		Petr Šulc, Alexander Solovyov, Sajid A. Marhon, Siyu Sun,  John LaCava, Omar Abdel-Wahab, Nicolas Vabret, Daniel D. De Carvalho, Rémi Monasson, Simona Cocco, Benjamin D. Greenbaum	

	<li><a href="https://pubs.acs.org/doi/full/10.1021/acssynbio.1c00336"> Kinetics of RNA and RNA:DNA hybrid strand displacement
	</a>  <br>
		ACS Synthetic Biology (2021) <br>
		H. Liu, F. Hong, F. Smith, J. Goertz, T. E. Ouldridge, M. Stevens, H. Yan,  P. Šulc	

	

	<li><a href="https://pubs.acs.org/doi/abs/10.1021/jacs.1c06226"> A Self-Regulating DNA Rotaxane Linear Actuator Driven by Chemical Energy </a>  <br>
		J. Am. Chem. Soc. 143. 13292 (2021) <br>
		Z. Yu, M. Centola, J. Valero, M. Matthies, P. Šulc, M. Famulok

	<li><a href="https://doi.org/10.1093/nar/gkab324"> OxDNA.org: a public webserver for coarse-grained simulations of DNA and RNA nanostructures </a>  <br>
		Nucleic Acids Research, (2021) <br>
	 	Erik Poppleton, Roger Romero, Aatmik Mallya, Lorenzo Rovigatti, Petr Šulc	

		 
	<li><a href="https://www.mdpi.com/2076-3417/11/6/2802"> DNA Nanodevices as Mechanical Probes of Protein Structure and Function	</a>  <br>
		Applied Sciences 11 (6), 2802 (2021) <br>
		 Nicholas Stephanopoulos, Petr Šulc	

	<li><a href="https://pubs.rsc.org/en/content/articlehtml/2021/sm/d0sm01639j">Coarse-grained nucleic acid–protein model for hybrid nanotechnology</a>  <br>
		Soft Matter (2021) ; arXiv <a href="https://arxiv.org/pdf/2009.09589">preprint</a><br>
	 	Jonah Procyk, Erik Poppleton, Petr Šulc	

		
	<li><a href="https://academic.oup.com/mbe/advance-article/doi/10.1093/molbev/msab036/6130826"> The heterogeneous landscape and early evolution of pathogen-associated CpG dinucleotides in SARS-CoV-2</a>  <br>
			Molecular Biology and Evolution (2021) <br>
			Andrea Di Gioacchino, Petr Šulc, Anastassia V. Komarova, Benjamin D. Greenbaum, Rémi Monasson, Simona Cocco
	
	

	<li><a href="https://www.frontiersin.org/articles/10.3389/fmolb.2021.693710/full"> A primer on the oxDNA model of DNA: When to use it, how to simulate it and how to interpret the results	</a>  <br>
		Front. Mol. Biosci., 17 June 2021 (2021) <br>
				A. Sengar, T. E. Ouldridge, O. Henrich, L. Rovigatti, P. Šulc	
	

	<li><a href="https://doi.org/10.1093/molbev/msab280"> Phenotype bias determines how RNA structures occupy the morphospace of all possible shapes</a>  <br>
		Molecular Biology and Evolution (2021) <br>
			K. Dingle, F. Ghaddar, P. Šulc, A.A. Louis	

	

	
	<h3> 2020</h3>


	<li><a href="https://journals.aps.org/prl/abstract/10.1103/PhysRevLett.125.118003"> Designing patchy interactions to self-assemble arbitrary structures	</a>  <br>
		Phys. Rev. Lett., Vol. 125, 118003,  (2020) ; arXiv <a href="https://arxiv.org/abs/2007.15873">preprint</a><br>
	 	Flavio Romano, John Russo, Lukáš Kroc, Petr Šulc	

	<li><a href="https://www.nature.com/articles/s41557-020-0539-8"> Meta-DNA structures</a>  <br>
		Nature Chemistry (2020) <br>
		Guangbao Yao, Fei Zhang, Fei Wang, Tianhuan Peng, Hao Liu, Erik Poppleton, Petr Šulc, Shuoxing Jiang, Lan Liu, Chen Gong, Xinxin Jing, Xiaoguo Liu, Lihua Wang, Yan Liu, Chunhai Fan, Hao Yan	

	<li><a href="https://dx.doi.org/10.1016%2Fj.bpj.2020.08.026"> The multiscale future of RNA modeling	</a>  <br>
		Biophysical Journal (2020) <br>
		 Petr Šulc	
	
	<li><a href="https://doi.org/10.1093/nar/gkaa854"> Understanding DNA interactions in crowded environments with a coarse-grained model</a>  <br>
		Nucleic Acids Research, Vol. 48, 19 (2020) <br>
	 	Fan Hong, John Schreck, Petr Šulc	

	<li><a href="https://doi.org/10.1093/nar/gkaa417">Design, optimization, and analysis of large DNA and RNA nanostructures through interactive visualization, editing, and molecular simulation </a>  <br>
		Nucleic Acids Research, Volume 48, Issue 12, (2020) <br>
	 	Erik Poppleton, Joakim Bohlin, Michael Matthies, Shuchi Sharma, Fei Zhang, Petr Šulc	


	
	<h3> 2019</h3>


	<li><a href="https://onlinelibrary.wiley.com/doi/abs/10.1002/jcc.26029"> TacoxDNA: A user‐friendly web server for simulations of complex DNA structures, from single strands to origami
	</a><br> Journal of computational chemistry 40 (29), 2586-2595	 (2019); article was featured on the front <a href="https://onlinelibrary.wiley.com/toc/1096987x/2019/40/29">cover</a> <br>
	Antonio Suma, Erik Poppleton, Michael Matthies, Petr Šulc, Flavio Romano, Ard A Louis, Jonathan PK Doye, Cristian Micheletti, Lorenzo Rovigatti
	
	<li><a href="https://www.sciencedirect.com/science/article/pii/S1047847719301340">An emergent understanding of strand displacement in RNA biology
	</a><br> Journal of Structural Biology 207 (3), 241-249	 (2019) <br>
	Fan Hong, Petr Šulc
	<li><a href="https://pubs.acs.org/doi/abs/10.1021/acsnano.8b08009">Triangulated Wireframe Structures Assembled Using Single-Stranded DNA Tiles
	</a><br> ACS nano 13 (2), 1839-1848	 (2019) <br>
	Michael Matthies, Nayan P Agarwal, Erik Poppleton, Foram M Joshi, Petr Šulc, Thorsten L Schmidt


	
	<h3> 2018 </h3>

	<li><a href="https://pubs.acs.org/doi/abs/10.1021/jacs.8b07180">Layered-crossover tiles with precisely tunable angles for 2D and 3D DNA crystal engineering
	</a><br> Journal of the American Chemical Society 140 (44), 14670-14676	(2018) <br>
	Fan Hong, Shuoxing Jiang, Xiang Lan, Raghu Pradeep Narayanan, Petr Šulc, Fei Zhang, Yan Liu, Hao Yan <br>
	</li>
	<li><a href="https://onlinelibrary.wiley.com/doi/abs/10.1002/anie.201804264">Rapid Photoactuation of a DNA nanostructure using an internal photocaged trigger strand
	</a><br> Angewandte Chemie International Edition 57 (30), 9341-9345	 (2018) <br>
	M Liu, S Jiang, O Loza, NE Fahmi, P Šulc, N Stephanopoulos
	</li>

	<h3> prior 2018 </h3>
<li><a href="https://www.nature.com/articles/nnano.2016.277">Self-assembling DNA nanotubes to connect molecular landmarks<br>
</a> Nature Nanotechnology, 2017<br>AM Mohammed, P Šulc, J Zenk, R Schulman
<li><a href="http://arxiv.org/abs/1506.02539">Coarse-grained modelling of supercoiled RNA</a><br>  
preprint: <a href="http://arxiv.org/abs/1506.02539">arXiv:1506.02539</a><br>C. Matek, P. Šulc, F. Randisi, J.P.K. Doye, A. A. Louis
<li><a href="http://scitation.aip.org/content/aip/journal/jcp/142/23/10.1063/1.4921957">Introducing Improved Structural Properties and Salt Dependence into a Coarse-Grained Model of DNA</a><br> 
J. Chem. Phys., 142, 234901 (2015) (article featured on the journal <a href="http://doye.chem.ox.ac.uk/news/jcp15b_cover.jpg">cover</a>)
, preprint: <a href="http://arxiv.org/abs/1504.00821">arXiv:1504.00821</a><br>B. E. K. Snodin, F. Randisi, M. Mosayebi, P. Šulc, J. S. Schreck, F. Romano, T. E. Ouldridge, R. Tsukanov, E. Nir, A. A. Louis, J. P. K. Doye
<li> <a href="http://dx.doi.org/10.1016/j.bpj.2015.01.023">Modelling toehold-mediated RNA strand displacement</a><br> Biophys. J. 108, iss. 5, 1238-1247 (2015) (article featured on the front <a href="http://doye.chem.ox.ac.uk/news/bj15_cover.jpg">cover</a>), preprint: <a href="http://arxiv.org/abs/1411.3239">arXiv:1411.3239</a><br> P. Šulc, T. E. Ouldridge, F. Romano, J. P. K. Doye, A. A. Louis 
<li> <a href="http://nar.oxfordjournals.org/content/early/2015/06/08/nar.gkv582">DNA hairpins primarily promote duplex melting rather than inhibiting hybridization</a><br>
Nucleic Acids Res. (2015) <br> preprint: <a href="http://arxiv.org/abs/1408.4401">arXiv:1408.4401</a><br> J. S. Schreck, T. E. Ouldridge, F. Romano, P. Šulc, L. Shaw, A. A. Louis, J. P. K. Doye 
<li> <a href="http://scitation.aip.org/content/aip/journal/jcp/140/23/10.1063/1.4881424">A nucleotide-level coarse-grained model of RNA</a><br>J. Chem. Phys. 140, 235102 (2014) (article featured on the front <a href="http://doye.chem.ox.ac.uk/news/jcp14_cover.jpg">cover</a>), preprint: <a href="http://arxiv.org/abs/1403.4180">arXiv:1403.4180</a><br> P. Šulc,  F. Romano,  T. E. Ouldridge, J. P. K. Doye, A. A. Louis 
<li> <a href="http://onlinelibrary.wiley.com/enhanced/doi/10.1002/jcc.23763">A comparison between parallelization approaches in molecular dynamics simulations on GPUs</a> <br>J. Comp. Chem 36 (2015), (article featured on the front <a href="https://dna.physics.ox.ac.uk/images/thumb/9/9f/Jcc_cover.png/424px-Jcc_cover.png">cover</a>) preprint: <a href="http://arxiv.org/abs/1401.4350">arXiv:1401.4350</a><br> L. Rovigatti, P. Šulc, István Z. Reguly, F. Romano 
<li> <a href="http://ieeexplore.ieee.org/xpl/articleDetails.jsp?arnumber=6963439"> Optimal Distributed Control of Reactive Power via the Alternating Direction Method of Multipliers</a><br> IEEE Transactions on Energy Conversion 29 (2014), preprint: <a href="http://arxiv.org/abs/1310.5748">arXiv:1310.5748</a><br> P. Šulc, S. Backhaus, M. Chertkov 
<li> <a href="http://pubs.rsc.org/en/Content/ArticleLanding/2013/CP/C3CP53545B#!divAbstract">Coarse-graining DNA for simulations of DNA nanotechnology </a><br> Phys. Chem. Chem. Phys. (2013), preprint: <a href="http://arxiv.org/abs/1308.3843">arxiv:1308.3843</a> <br> J.P.K. Doye, T. E. Ouldridge, A. A. Louis, F. Romano, P. Šulc, C. Matek, B.E.K. Snodin, L. Rovigatti, J. S. Schreck, R.M. Harrison, W.P.J. Smith
<li> <a href="http://nar.oxfordjournals.org/content/41/22/10641">On the biophysics and kinetics of toehold-mediated DNA strand displacement</a> <br> Nucleic Acids Res. (2013) <br> N. Srinivas, T.E. Ouldridge, P. Šulc, J.M. Schaeffer, B. Yurke, A.A. Louis, J.P.K. Doye, E. Winfree
<li>  <a href="http://nar.oxfordjournals.org/content/early/2013/08/08/nar.gkt687"> DNA hybridization kinetics: zippering, internal displacement and sequence dependence</a> <br> Nucleic Acids Res. (2013), preprint: <a href="http://arxiv.org/abs/1303.3370">arxiv:1303.3370</a> <br> T.E. Ouldridge, P. Šulc, F. Romano, J.P.K. Doye, A.A. Louis <br>
<li>  <a href="http://link.springer.com/article/10.1007%2Fs11047-013-9391-8"> Simulating a burnt-bridges DNA motor with a coarse-grained DNA model</a> <br>  Natural Computing (2014), preprint: <a href="http://arxiv.org/abs/1212.4536">arxiv:1212.4536</a> <br>  P. Šulc, T.E. Ouldridge, F. Romano, J.P.K. Doye, A.A. Louis <br>
<li> <a href="http://jcp.aip.org/resource/1/jcpsa6/v137/i13/p135101_s1"> Sequence-dependent thermodynamics of a coarse-grained DNA model</a> <br> J. Chem. Phys. (2012), (article featured on the front <a href="http://doye.chem.ox.ac.uk/news/jcp12d_cover_h200.jpg">cover</a>) preprint: <a href="http://arxiv.org/abs/1207.3391">arxiv:1207.3391</a> <br>  P. Šulc, F. Romano, T.E. Ouldridge, L. Rovigatti, J.P.K. Doye, A.A. Louis 
<li>  <a href="http://link.aps.org/doi/10.1103/PhysRevE.85.051905"> Non-Gaussianity in single-particle tracking: Use of kurtosis to learn the characteristics of a cage-type potential</a> <br>  Phys. Rev. E (2012), preprint: <a href="http://arxiv.org/abs/1102.2290">arXiv:1102.2290</a> <br> P. Lushnikov, P. Šulc, K. Turitsyn
<li> <a href="http://dx.doi.org/10.1109/JPROC.2011.2116750"> Options for Control of Reactive Power by Distributed Photovoltaic Generators</a> <br> Proceedings of the IEEE (2011), preprint: <a href="http://arxiv.org/abs/1008.0878">arXiv:1008.0878</a><br> K. Turitsyn, P. Šulc, S. Backhaus, M. Chertkov 
<li> <a href="http://dx.doi.org/10.1109/SMARTGRID.2010.5622021"> Local Control of Reactive Power by Distributed Photovoltaic Generators</a> <br> IEEE SmartGridComm 2010, preprint: <a href="http://arxiv.org/abs/1006.0160">arXiv:1006.0160</a> <br> K. Turitsyn, P. Šulc, S. Backhaus, M. Chertkov 
<li>  <a href="http://iopscience.iop.org/1751-8121/43/28/285003"> Belief propagation for graph partitioning</a> <br> Journal of Physics A: Math. &amp; Theor. (2010), preprint: <a href="http://arxiv.org/abs/0912.3563">arXiv:0912.3563</a> <br> P. Šulc, L. Zdeborová
<li> <a href="http://dx.doi.org/10.1109/PES.2010.5589663"> Distributed control of reactive power flow in a radial distribution circuit with high photovoltaic penetration</a> <br>  IEEE Power and Energy Society General Meeting 2010, preprint: <a href="http://arxiv.org/abs/0912.3281">arXiv:0912.3281</a> <br>K. Turitsyn, P. Šulc, S. Backhaus, M. Chertkov 
<li> <a href="http://www.ieu.uzh.ch/wagner/papers/Sulc_JCBC_2010.pdf"> Quantifying Slow Evolutionary Dynamics in RNA Fitness Landscapes</a> <br> Journal of Bioinformatics and Computational Biology, (2010), preprint:  <a href="http://arxiv.org/abs/0911.5366">arXiv:0911.5366</a> <br>  P. Šulc, O. C. Martin, A. Wagner 
<li> <a href="http://pre.aps.org/abstract/PRE/v81/i3/e031111"> Random walk return probabilities and hitting times on sparse Erdos-Renyi graphs</a><br> Phys. Rev. E 81, (2010), preprint:  <a href="http://arxiv.org/abs/0902.2173">arXiv:0902.2173</a><br> O. C. Martin, P. Šulc
<li> <a href="http://iopscience.iop.org/1751-8121/40/50/013/"> Group theoretical construction
 of mutually unbiased bases in Hilbert spaces of prime dimensions</a> <br> Journal of Physics A: Math. &amp; Theor. (2007), preprint: <a href="http://arxiv.org/abs/0708.4114">arXiv:0708.4114 </a><br> P. Šulc, J. Tolar
</a>
</ol>
<h2>Book Chapters:</h2>
<ol>
	<li> <a href="myimages/chapter.pdf">Introduction to Molecular simulation</a> <br> "<a href="https://mitpress.mit.edu/books/quantitative-biology">Quantitative Biology: Theory, Computational Methods and Models</a>", ed. B. Munsky, W.S. Hlavacek and L.S. Tsimring <br> MIT Press, 2018, pp. 179-208 <br>
		Petr Sulc, Jonathan P.K. Doye, and Ard A. Louis
	</li>
	<li> <a href="https://link.springer.com/protocol/10.1007/978-1-0716-3028-0_6">The oxDNA coarse-grained model as a tool to simulate DNA origami</a> <br> 
		DNA and RNA Origami: Methods and Protocols, Methods in Molecular Biology, Humana Press, (2023), preprint:  <a href="https://arxiv.org/pdf/2004.05052"> arxiv</a> <br>
		Jonathan PK Doye, Hannah Fowler, Domen Prešern, Joakim Bohlin, Lorenzo Rovigatti, Flavio Romano, Petr Šulc, Chak Kui Wong, Ard A Louis, John S Schreck, Megan C Engel, Michael Matthies, Erik Benson, Erik Poppleton, Benedict EK Snodin

	</li>
</ol>
</div>

</div>


</body>
</html>
