<?xml version="1.0"?><!DOCTYPE html PUBLIC "-//W3C//DTD XHTML+ARIA 1.0//EN" "http://www.w3.org/MarkUp/DTD/xhtml-aria-1.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en-GB">
<head>
		<title>Homepage of Petr Šulc</title>
		<meta name="description" content="" />
		<meta name="keywords" content="" />
		<meta http-equiv="content-language" content="en-GB" />
		<meta http-equiv="content-type" content="text/html; charset=utf-8" />
		<!--<link rel="shortcut icon" href="/favicon.ico" />-->

		<meta http-equiv="content-script-type" content="text/javascript" />
		<meta http-equiv="content-style-type" content="text/css" />
		<meta http-equiv="imagetoolbar" content="no" />
        <meta http-equiv="Cache-Control" content="no-cache, no-store, must-revalidate">
        <meta http-equiv="Pragma" content="no-cache">
        <meta http-equiv="Expires" content="0">

		<meta name="robots" content="index,follow" />


        <meta http-equiv="content-script-type" content="text/javascript" />
        <meta http-equiv="content-style-type" content="text/css" />
        <meta http-equiv="imagetoolbar" content="no" />

        <meta name="copyright" content="&copy; 2010" />

		  <link rel="stylesheet" type="text/css" href="css/smoothness/jquery-ui-1.8.5.custom.css" />
        <link rel="stylesheet" type="text/css" href="css/main.css" />

        <script type="text/javascript" src="js/jquery-1.4.2.min.js"></script>
        <script type="text/javascript" src="js/jquery-ui-1.8.5.custom.min.js"></script>

        <script type="text/javascript">

        $(document).ready(function(){
           $('button.btp').button({
            icons: {primary: "ui-icon-arrowthickstop-1-n"}
            });

        });


        </script>
</head>

<body>

<p class="ui-helper-hidden-accessible">
<a href="#content-main" title="main content">main content</a>
</p>

<div id="wrapper-outer">

	<div id="header">

	<div id="tr-image">
	 <img src="images/logo.png" height="150" alt="Photograph"  width="120.5"  align=top  border="0" />
	
	</div>



	<h1>Petr Šulc's lab</h1>

	<div id="menu-top">
		<ul>
    <li class="generic"><a href="index.html">Home</a></li>
    <li class="generic"><a href="group.html">Members</a></li>
    <li class="generic inner current"><a href="research.html">Research</a></li>
    <li class="generic"><a href="software.html">Software</a></li>
    <li class="generic"><a href="publications.html">Publications</a></li>
    <li class="generic"><a href="bio.html">Petr Šulc</a></li>
    </ul>
	</div>

	</div>


	<div id="content-main">



<h2>Research</h2>
This is a brief overview of our research topics. Interested prospective students or postdocs should <a href="bio.html">contact</a> us directly for the most up-to-date list of research opportunities. Our research areas are at the interface of bionanotechnology, physical chemistry, biophysics, soft matter physics, and synthetic biology. 
In studying these topics, we combine approaches from physics, bioinformatics, bioengineering, biochemistry and computer science. We offer both theory-only and experiment-only projects,
as well as combination of both.


<h3> Bionanotechnology: Modeling-driven experiments to build structures and devices out of DNA, RNA and proteins </h3>
Bionanotechnology uses molecules as basic building blocks to construct nanoscale devices. We work mostly in DNA and RNA nanotechnology, as well as in DNA-protein hybrid nanostructure design.  DNA nanotechnology is a rapidly developing field which uses DNA molecules as basic building blocks to assemble nanoscale structures
as well as active molecular devices. Computer simulations using our coarse-grained model provide insight into the functioning of such systems and give us better understanding of underlying physical processes.
 Our approach is model-driven, where we develop multiscale models 
that guide experimental efforts carried out either in our lab or in collaboration with other colleagues. 
Many of our projects harness coarse-grained models of nucleic acids. Our DNA/RNA model, oxDNA and oxRNA, is parametrized to reproduce quantitatively structural, mechanical and thermodynamic
 properties of DNA and RNA. It has been successfully used to study both fundamental physics of DNA/RNA molecules as well 
as nucleic acid nanotechnological systems.  Depending on the project, the experiments are either carried out in our group and with a collaborator. The currently available projects in this topic can involve just modeling, just experiment, or combination of both, depending on the expertise and wishes of the student.

Our current DNA/RNA nanotechnology projects (both on theory and experiment level) comprise two main directions, one is more material science-oriented, the other is more towards biological systems:
<ul>
 <li> New self-assembled biomaterials development and nanofabrication, combining multiscale 
simulation with experimental studies. These materials have promising applications in e.g. construction of photonic devices and nanoelectronics.
<li> Biomedical application: Development of new nanostructures / nanodevices for biomedical applications for diagnostic or therapeutics. These include targeted drug delivery, improving RNA vaccines and diagnostic tools.
</ul>
<b>Selected articles:</b>
<ol>
    <li><a href="https://www.science.org/stoken/author-tokens/ST-1861/full"> Inverse design of a pyrochlore lattice of DNA origami through model-driven experiments
</a>  <br>
Science, 384, 6697, (741-742), (2024); <br>
Hao Liu, Michael Matthies, John Russo, Lorenzo Rovigatti, Raghu Pradeep Narayanan, Thong Diep, Daniel McKeen, Oleg Gang, Nicholas Stephanopoulos, Francesco Sciortino, Hao Yan, Flavio Romano, Petr Šulc


	<li><a href="https://doi.org/10.1039/D2NR03533B"> A simple solution to the problem of self-assembling cubic diamond crystals	</a>  <br>
		Nanoscale 14 (2022) <br>
		Lorenzo Rovigatti, John Russo, Flavio Romano, Michael Matthies, Lukáš Kroc, Petr Šulc


<li><a href="https://pubs.acs.org/doi/full/10.1021/acsnano.4c10796"> Hairygami: Analysis of DNA Nanostructures' Conformational Change Driven by Functionalizable Overhangs	</a>  <br>
	ACS Nano, 2024 <br>
	Matthew Sample, Thong Diep, Hao Liu, Michael Matthies, Petr Šulc


<li><a href="https://www.nature.com/articles/s41565-023-01516-x"> A rhythmically pulsing leaf-spring nanoengine that drives a passive follower
</a>  <br>
 Nature Nanotechnology 19, 226-236 (2024) ; biorxiv <a href="https://www.biorxiv.org/content/10.1101/2021.12.22.473833v1"> preprint  </a><br>
M. Centola, E. Poppleton, M. Centola, J. Valero, S. Ray, R. Welty, Nils G. Walter, P. Šulc, M. Famulok


<li><a href="https://pubs.acs.org/doi/10.1021/jacs.3c07491"> CytoDirect: A Nucleic Acid Nanodevice for Specific and Efficient Delivery of Functional Payloads to the Cytoplasm
</a>  <br>
	JACS (2023) <br>
	Lu Yu, Yang Xu, Md Al-Amin, Shuoxing Jiang, Matthew Sample, Abhay Prasad, Nicholas Stephanopoulos, Petr Šulc, and Hao Yan

<li> <a href="https://www.biorxiv.org/content/10.1101/2023.09.18.558353v1.abstract">High-affinity binding to the SARS-CoV-2 spike trimer by a nanostructured, trivalent protein-DNA synthetic antibody
</a> <br>
biorxiv preprint  <br>
Yang Xu, Rong Zheng, Abhay Prasad, Minghui Liu, Zijian Wan, Xiaoyan Zhou, Ryan M Porter, Matthew Sample, Erik Poppleton, Jonah Procyk, Hao Liu, Yize Li, Shaopeng Wang, Hao Yan, Petr Šulc, Nicholas Stephanopoulos


</ol> 


<h3> Understanding the principles of self-assembly </h3>
How to design building blocks so that they reliably assemble into a target structure? Biology can achieve structure of extraordinary complexity by self-assembly of different building block,
 something that nanotechnology has been trying to mimic for a long time. We employ methods of computer simulations and statistical physics to answer questions such as "What is the minimal number of blocks needed to achieve a target structure", and "How to avoid possible kinetic traps and undesired assemblies?", 
 "How to periodic crystals assemble?". This work is at the interface of computational physics, chemistry and soft matter and the projects in this area typically involve coding (Python / C++) and studies of assembly processes using high performance computing and custom-written codes, as well as soft matter
 physics theory.
 
 <br>
 <b>Selected articles:</b>
<ol>

	<li><a href="https://journals.aps.org/prl/abstract/10.1103/PhysRevLett.125.118003"> Designing patchy interactions to self-assemble arbitrary structures	</a>  <br>
		Phys. Rev. Lett., Vol. 125, 118003,  (2020) ; <br>
	 	Flavio Romano, John Russo, Lukáš Kroc, Petr Šulc	

	<li><a href="https://pubs.acs.org/doi/10.1021/acsnano.2c09677"> Designing the self-assembly of arbitrary shapes using minimal complexity building blocks	</a>  <br> 
		ACS Nano (2023)  ;	 <br>
			Joakim Bohlin, Andrew Turberfield, Ard Louis, Petr Šulc
	

	<li><a href="https://doi.org/10.1073/pnas.2219458120"> Design strategies for the self-assembly of polyhedral shells	</a>  <br>
		Proceedings of the National Academy of Sciences 120 (16),  (2023)  <br>
		Diogo E. P. Pinto; Petr Šulc; Francesco Sciortino; John Russo


	<li><a href="https://iopscience.iop.org/article/10.1088/1361-648X/ac5479/meta"> SAT-assembly: A new approach for designing self-assembling systems 	</a>  <br>
		Journal of Physics: Cond. Matter (2022);  <br>
	 John Russo, Flavio Romano, Lukáš Kroc, Francesco Sciortino, Lorenzo Rovigatti, Petr Šulc	


<li><a href="https://journals.aps.org/prxlife/abstract/10.1103/PRXLife.2.013012"> RNA-induced allosteric coupling drives viral capsid assembly  </a>  <br>
	PRX Life (2024)  <br>
	Sean Hamilton, Tushar Modi, Petr Šulc, Banu Ozkan

	
</ol>

 <h3> Machine learning for sequence ensembles </h3>
 Aptamers are short DNA or RNA molecules selected by in-vitro protocol (SELEX) to bind to a certain molecular target (protein, small molecule, specific cell surface etc.) with 
 applications for diagnostics and therapeutics. We use machine learning to study the sequence ensembles (DNA or RNA aptamers or short peptides selected to bind to a certain target), identify functional motifs and generate in-silico new binders. As another example of our work in this area,
  we use machine learning and statistical physics methods to quantify immunostimulatory motifs in RNA transcripts presents viral genomes, as well as in tissues, with applications to diagnostics, treatment as well as bionanotechnology.

 <br>
 <b>Selected articles:</b>
<ol>
    <li> <a href="https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1010561"> Generative and interpretable machine learning for aptamer design and analysis of in vitro sequence selection
	</a>  <br>
	PLOS Comp. Biology (2022);  <br>
	Andrea Di Gioacchino,  Jonah Procyk,  Marco Molari,  John S. Schreck, Yu Zhou, Yan Liu,  Rémi Monasson,  Simona Cocco, Petr Šulc
    </li>

<li><a href="https://pubs.acs.org/doi/10.1021/acscentsci.4c01428">Combination of Coevolutionary Information and Supervised Learning Enables Generation of Cyclic Peptide Inhibitors with Enhanced Potency from a Small Data Set	</a>  <br>
	ACS Central Science, 2024 <br>
	Ylenia Mazzocato, Nicola Frasson, Matthew Sample, Cristian Fregonese, Angela Pavan, Alberto Caregnato, Marta Simeoni, Alessandro Scarso, Laura Cendron, Petr Šulc, Alessandro Angelini

	<li><a href="https://www.cell.com/cell-genomics/fulltext/S2666-979X(25)00267-8"> Repeats Mimic Immunostimulatory Viral Features Across a Vast Evolutionary Landscape
		</a>  <br>
		Cell Genomics (2025)  <br>
		P. Šulc, A. Di Gioacchino, A. Solovyov, S. Sun, S. Martis, S. Marhon, H. Lindholm, R. Chen, A. Hosseini, H. Jiang, B. Ly, M. Taylor, P. Mehdipour, O. Abdel-Wahab, N. Rusk, N. Vabret, J. LaCava, D. De Carvalho, R. Monasson, S. Cocco, B. Greenbaum
</ol>

  <h3> Biophysics of DNA and RNA</h3>
  Our modeling efforts, primarily aimed at simulations of DNA/RNA in nanotechnology context, are also used to understand basic biophysics of DNA and RNA, including processes such as hybridization and strand displacement. Understanding these processes, besides being relevant for molecular computing and 
  understanding of DNA/RNA function, also provide insights into biological complexes, such as CRISPR-Cas systems. We often used our models such as oxDNA, oxRNA and oxNA to use coarse-grained simulations to gain insight into particular properties of the molecules and relate them to experimental observations.

 <br>
<b>Selected articles:</b> 
<ol>
	<li><a href="https://doi.org/10.1093/nar/gkaa854"> Understanding DNA interactions in crowded environments with a coarse-grained model</a>  <br>
		Nucleic Acids Research, Vol. 48, 19 (2020) <br>
	 	Fan Hong, John Schreck, Petr Šulc	
    <li><a href="https://www.pnas.org/doi/10.1073/pnas.2416988122">Controlling DNA-RNA strand displacement kinetics with base distribution</a>  <br>
		PNAS (2025);   <br>
		Eryk Ratajczyk, Jonathan Bath, Petr Sulc, Jonathan Doye, Ard Louis, Andrew Turberfield
<li><a href="https://www.nature.com/articles/s41467-024-51813-9"> Single-Molecule Force Spectroscopy of Toehold-Mediated Strand Displacement
</a>  <br>
	Nature Communications 15 (2024)<br>
	Andreas Walbrun, Tianhe Wang, Michael Matthies, Petr Šulc, Friedrich Simmel, Matthias Rief
    <li> <a href="http://dx.doi.org/10.1016/j.bpj.2015.01.023">Modelling toehold-mediated RNA strand displacement</a><br> Biophys. J. 108, iss. 5, 1238-1247 (2015) <br> P. Šulc, T. E. Ouldridge, F. Romano, J. P. K. Doye, A. A. Louis 

</ol>


<h3>Model and method development for DNA and RNA simulations</h3>
Most of the models and simulation methods that we develop are available within the context of the oxDNA modeling ecosystem, centered around open source simulation code oxDNA. A lot of our work is enabled by development and parameterization of these models, as well as development of novel methods to enable enhanced sampling
or large complex structures.

 <br>
 <b>Selected articles:</b>
 <ol>

    	<li><a href="https://pubs.rsc.org/en/content/articlehtml/2021/sm/d0sm01639j">Coarse-grained nucleic acid–protein model for hybrid nanotechnology</a>  <br>
		Soft Matter (2021) <br>
	 	Jonah Procyk, Erik Poppleton, Petr Šulc	


	<li><a href="https://joss.theoj.org/papers/10.21105/joss.04693.pdf"> oxDNA: coarse-grained simulations of nucleic acids made simple	</a>  <br>
			Journal of Open Source Software (2023)  <br>
			Erik Poppleton, Michael Matthies, Debesh Mandal, Flavio Romano, Petr Šulc, Lorenzo Rovigatti

<li><a href="https://pubs.aip.org/aip/jcp/article/160/11/115101/3275728/Coarse-grained-modeling-of-DNA-RNA-hybrids?searchresult=1"> Coarse-grained modelling of DNA-RNA hybrids</a>  <br>
	J. Chem. Phys. 160, 115101 (2024);  <br>
		Eryk J. Ratajczyk, Petr Šulc, Andrew J. Turberfield, Jonathan P.K. Doye, Ard A. Louis
<li> <a href="http://scitation.aip.org/content/aip/journal/jcp/140/23/10.1063/1.4881424">A nucleotide-level coarse-grained model of RNA</a><br>J. Chem. Phys. 140, 235102 (2014) <br> P. Šulc,  F. Romano,  T. E. Ouldridge, J. P. K. Doye, A. A. Louis 

 <li> <a href="http://jcp.aip.org/resource/1/jcpsa6/v137/i13/p135101_s1"> Sequence-dependent thermodynamics of a coarse-grained DNA model</a> <br> J. Chem. Phys. (2012),  <br>  P. Šulc, F. Romano, T.E. Ouldridge, L. Rovigatti, J.P.K. Doye, A.A. Louis 
</li>
 </ol>


<h3>Infrastructure for DNA/RNA nanotechnology</h3>
We develop and maintain popular highly used tools and services for the nucleic acid nanotechnology and molecular programming community. These include public webserver nanobase.org, a repository for sharing nanostructure design, free simulation service oxDNA.org, and modeling/editing/design/interactive simulation
tool oxView.  We have numerous projects (at undergraduate, master's and graduate level) available in design, extension, and further development of these services, typically available at different skills of Linux system administration, database, AI-prompting and coding skills.  

 <br>
 <b>Selected articles:</b>
 <ol>

	<li><a href="https://www.nature.com/articles/s41596-022-00688-5"> Design and simulation of DNA, RNA and hybrid protein–nucleic acid nanostructures with oxView	</a>  <br>
		Nature Protocols (2022)  <br>
		Joakim Bohlin, Michael Matthies, Erik Poppleton, Jonah Procyk, Aatmik Mallya, Hao Yan, Petr Šulc
	
	<li><a href="https://doi.org/10.1093/nar/gkaa417">Design, optimization, and analysis of large DNA and RNA nanostructures through interactive visualization, editing, and molecular simulation </a>  <br>
		Nucleic Acids Research, Volume 48, Issue 12, (2020) <br>
	 	Erik Poppleton, Joakim Bohlin, Michael Matthies, Shuchi Sharma, Fei Zhang, Petr Šulc	

        <li><a href="https://doi.org/10.1093/nar/gkab1000"> Nanobase.org: a repository for DNA and RNA nanostructures </a>  <br>
		Nucleic Acids Research, (2022)<br>
		Erik Poppleton, Aatmik Mallya, Swarup Dey, Joel Joseph, Petr Šulc

        <li><a href="https://doi.org/10.1093/nar/gkab324"> OxDNA.org: a public webserver for coarse-grained simulations of DNA and RNA nanostructures </a>  <br>
		Nucleic Acids Research, (2021) <br>
	 	Erik Poppleton, Roger Romero, Aatmik Mallya, Lorenzo Rovigatti, Petr Šulc	
    </ol>

<h3>Smartgrid: Control of reactive power from photovoltaic generators</h3>
Smartgrid refers to power grid which incorporates modern computer communication technology to improve efficiency and reliability of production, distribution and consumption of electricity. One of the big challenges that the power grid faces at the moment is the integration of renewable sources of energy. <br> 
In prior work we investigated (<a href="http://dx.doi.org/10.1109/JPROC.2011.2116750">[1]</a>, <a href="http://dx.doi.org/10.1109/SMARTGRID.2010.5622021">[2]</a>, <a href="http://dx.doi.org/10.1109/PES.2010.5589663">[3]</a>) the possibilities of controlling reactive power flow in a distribution network that contains photovoltaic generators (typical example would be an urban neighborhood with houses with photovoltaic cells on roofs). We considered both global and local control of the generated reactive power and found optimization schemes that can reduce thermal losses in the distribution line and improve voltage stability.
This direction is currently not an active research topic in our lab.

<h2>Funding</h2>
Current and/or prior support from <a href="https://www.onr.navy.mil/">ONR</a>,  <a href="http://www.nsf.gov">National Science Foundation</a>,  <a href="https://www.energy.gov/">Department of Energy</a> , <a href="https://www.nih.gov"> NIH</a>, <a href="https://erc.europa.eu/homepage">ERC</a> is gratefully acknowledged. 


</div>

</div>


</body>
</html>
